GOVERNANCE.md

```diff
@@ -253,10 +255,9 @@ A TSC member has significant sway in software design decisions. For this reason,
 
 #### Nomination
 
-- To be nominated, a nominee is expected to already be active in technical discussions and performing some of the responsibilities of a TSC member.
-- You can be nominated by any existing Core member (L3 or above). Note: This includes all existing TSC members as well.
-- Once nominated, there will be a vote by existing Core members.
-- See [vote rules & requirements](#voting) for info on how this vote works.
+- The makeup of the TSC is decided by the project Steward.
```

Comment:
Maybe worth precising that the Steward can remove the TSC membership from any active TSC member if it doesn't align with the goals of the TSC, or if there's a better fit within the Core team. Technically this could be inferred from the sentence as it is written now but I think it could be useful to make it more explicit